UABEAvalonia/Forms/GameObjectViewWindow.axaml.cs

```diff
@@ -141,7 +141,7 @@ private void PopulateHierarchyTreeView()
                 return;
 
             // clear treeview
-            gameObjectTreeView.Items = new AvaloniaList<object>();
+            gameObjectTreeView.ItemsSource = new AvaloniaList<object>();
```

Comment:
`gameObjectTreeView.Items.Clear();`
We add to the items later on instead of assigning it, so we still need to use `Items`.